### PR TITLE
Address security vulnerabilities.

### PR DESCRIPTION
**Pull request by BrianBatchelder**
### What do these changes do?
This PR updates libraries with known security vulnerabilities to ones without known vulnerabilities. A dependency on `npm` was removed because it isn't necessary, and was exposing the plugin to some security vulnerabilities of its own.

### Why are these changes necessary?
Github detected, and `npm audit` confirmed, that our plugin contained libraries with known security vulnerabilities.

### How did you verify these changes?
Local `npm audit`. Will monitor GitHub after PR merge.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,5 @@
   },
   "license": "Apache-2.0",
   "typings": "./js/index.d.ts",
-  "dependencies": {
-    "npm": "^5.8.0"
```
Comment (marc-scig) [inline package.json]: do we still want the npm dependency?

Comment (BrianBatchelder) [reply package.json]: It isn't necessary, and was exposing us to some security vulnerabilities of its own.

Comment (rlepinski) [reply package.json]: No idea why it was in there.

**Comment by rlepinski** [comment]
👍

**Pull request merged**